tests/gtest/test_population_CatchNumbersAtAge.cpp

```diff
@@ -39,6 +39,8 @@ namespace
         // loop over fleets to get catch numbers at age for each fleet
         for (size_t fleet_index = 0; fleet_index < population.nfleets; fleet_index++)
         {
+          
+            if(!population.fleets[fleet_index]->is_survey){
```

Comment:
Suggestion: add an unit test to check catch_weight_at_age and catch_numbers_at_age from a survey fleet.

Reply:
We can add a test after refactoring the fleet module. See details [here](https://github.com/NOAA-FIMS/FIMS/issues/284#issuecomment-1623556774)